src/config.rs

```diff
@@ -484,7 +484,7 @@ pub struct Notifications {
     #[serde(default = "default_true")]
     pub show_message: bool,
     #[serde(default)]
-    pub play_sound: bool,
+    pub sound_hint: String,
```

Comment:
```suggestion
    pub sound_hint: Option<String>,
```
In rust, `None` is usually used to represent that a value is missing since there is a difference between an empty string and no string.